mlir/lib/Dialect/Vector/Utils/VectorUtils.cpp

```diff
@@ -317,3 +317,8 @@ SmallVector<OpFoldResult> vector::getMixedSizesXfer(bool hasTensorSemantics,
                          : memref::getMixedSizes(rewriter, loc, base);
   return mixedSourceDims;
 }
+
+bool vector::isLinearizableVector(VectorType type) {
+  auto numScalableDims = llvm::count(type.getScalableDims(), true);
+  return ((type.getRank() > 1) && (numScalableDims <= 1));
```

Comment:
I think we can drop the outermost `()` pair.

Reply:
Both the outer and inner parentheses are redundant here: `type.getRank() > 1 && numScalableDims <= 1`

Reply:
oh good catch! I did not notice that there are inner parentheses!

Reply:
I'd like to keep the inner parentheses - I find that easier to parse. Probably a subjective thing 😅